cmd/upgrade/cluster/cmd.go

```diff
@@ -215,7 +217,7 @@ func runWithRuntime(r *rosa.Runtime, cmd *cobra.Command) error {
 	}
 
 	// Check mandatory parameters and enable interactive mode if needed
-	if currentUpgradeScheduling.ScheduleDate == "" && currentUpgradeScheduling.ScheduleTime == "" &&
+	if (currentUpgradeScheduling.ScheduleDate == "" && currentUpgradeScheduling.ScheduleTime == "") &&
```

Comment:
Considering it's 'ands' why have the ()?

Reply:
Should this be an 'or' to match https://github.com/openshift/rosa/pull/1459/files#diff-3f508bee7428efffa6218f9a113aa7c939e1fc6dd8e30554b969aa1ccd7cc47aR142?

Reply:
it should be an or indeed! good catch thanks

Reply:
actually after Lucas comment, I fixed again the check as the case of no schedule specified is valid (and should schedule an update immediately)